### PR TITLE
added bikescontroller and view jbuilder

### PR DESCRIPTION
**Pull request by desho8**
@andrey-maleyev , added BikesController and View/Jbuilder, please check, thks.

### CODE REVIEW

File: app/controllers/api/v1/bikes_controller.rb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    @bike.update(bike_params)
+    # render json: @bike
+  end
+
+  def destroy
+    # @bike.destroy
+    # render json: @bike
+  end
+
+  # private
```
Comment (andrey-maleyev) [inline app/controllers/api/v1/bikes_controller.rb]: it should not be a comment

File: app/controllers/api/v1/bikes_controller.rb [hunk 2]
```diff
@@ -0,0 +1,37 @@
+class Api::V1::BikesController < Api::V1::BaseController
+  skip_before_action: verify_authencity_token, only: [:create, :update, :destroy]
+
```
Comment (andrey-maleyev) [inline app/controllers/api/v1/bikes_controller.rb]: add before_action get_bike method

File: app/controllers/api/v1/bikes_controller.rb [hunk 3]
```diff
@@ -0,0 +1,37 @@
+class Api::V1::BikesController < Api::V1::BaseController
+  skip_before_action: verify_authencity_token, only: [:create, :update, :destroy]
+
+  def index
+    @bike = Bikes.all
+    # render json: @bikes
+  end
+
+  def show
```
Comment (andrey-maleyev) [inline app/controllers/api/v1/bikes_controller.rb]: we need to delete (or comment) this method

File: app/controllers/api/v1/bikes_controller.rb (hunk 2 above)
Comment (desho8) [reply app/controllers/api/v1/bikes_controller.rb]: done.

File: app/controllers/api/v1/bikes_controller.rb (hunk 3 above)
Comment (desho8) [reply app/controllers/api/v1/bikes_controller.rb]: ok done.

File: app/controllers/api/v1/bikes_controller.rb (hunk 1 above)
Comment (desho8) [reply app/controllers/api/v1/bikes_controller.rb]: ok, i have removed that.

**Pull request merged**